res/games/formatters/faf.qthtml

```diff
@@ -9,8 +9,8 @@
 <font size="-1">by</font>&nbsp;<font color="{color}"><b>{host}</b></font></font>
 </td>
 <td align="right">
-<b><font color="silver" size="-1">Game Quality</font><br/>
-<font color="silver" size="+2">{gamequality}</font></b>
+<b><font color="silver" size="-1">Avg. Rating</font><br/>
+<font color="silver" size="+1">{avgrating}</font></b>
```

Comment:
I currently tent again more to size="+2" than "+1" .. but it's a taste thingy.
(same for mod.qthtml)